app/templates/base_root.html

```diff
@@ -1,11 +1,8 @@
 <html>
   <head>
 
-    {% if title %}
```

Comment:
What I meant earlier was, to change this block of code to

```
{% if title %}
<title>{{ title }}</title>
{% else %}
<title>Literapy</title>
{% endif %}
```

Reply:
If you want Literapy to be always in the title adapt the snippet as follows:

```
{% if title %}
<title>{{ title }} - Literapy</title>
{% else %}
<title>Literapy</title>
{% endif %}
```

Reply:
Your current code her introduces two problems, however

1. Except for `/templates/book_details.html` no other template uses the _title_ -slot. This means that the title on these pages will be "_ - Literapy_" right now.
2. For all the routes (in routes.py) that set the title property it won't show up now.